### DISCUSSION TITLE
How to make the badge automatically refer to the current repository?

### DISCUSSION THREAD

**Discussion by namasikanam** [community/community#28536] [Other Feature Feedback, Questions, & Ideas]
I am trying to make my badge of repository still work normally after forking. Now, I use a static link for my badge. But after forking, the badge of the newer repository still show the information of the older repository, which needs more manual modification.

**Comment by jfullstackdev** [comment]
can you send the link of that repo ? so we can take a look also

**Reply by namasikanam** [reply]
OK, I simply create a minimal working example [learn-badge](https://github.com/namasikanam/learn-badge).

**Reply by namasikanam** [reply]
You can fork it and push a new commit. The time badge in readme won't update, because it statically refer to my repo, but not the current (forked) repo.

**Reply by jfullstackdev** [reply]
this is when editing, 

![image](https://user-images.githubusercontent.com/47092464/183234221-ca63f7eb-2e4d-4b71-92cc-238e71b3cafa.png)

what are you trying to achieve by then ? it's already your username and repo name

**Reply by jfullstackdev** [reply]
> You can fork it and push a new commit. The time badge in readme won't update, because it statically refer to my repo, but not the current (forked) repo.

how long have passed since you updated this ? 

some cards and badges like this will take time to update

**Reply by namasikanam** [reply]
In my example, I expect the badge to show "the last update time of the current repository". I wish that, after you fork my repo, you change "hahahah" to "haha" and push this commit, then the badge will show the time that you push this commit. But now, the badge will not change.
